packages/components/src/RadarChart/RadarChart.component.js

```diff
@@ -136,10 +136,47 @@ Dot.propTypes = {
 	index: PropTypes.number,
 };
 
+/**
+ * This function provides a clickable axis dot element
+ * @param {Object} props the current props of the Radar
+ */
+// TODO 6.0: remove this export
```

Comment:
☝️

Reply:
Exports are required for compatibility until V6 no? A faulty assumption maybe.

Reply:
oh, sorry, i thought you forgot to remove it before committing